src/DropDownMenu/DropDownMenu.js

```diff
@@ -364,7 +365,8 @@ class DropDownMenu extends Component {
             {displayValue}
           </div>
           <IconButton
-            tabIndex={this.props.disabled ? -1 : undefined}
+            disabled={disabled}
+            tabIndex={disabled ? -1 : undefined}
```

Comment:
The `EnhancedButton` already has a logic to implement that. I think that you can remove this line.
```jsx
tabIndex: disabled || disableKeyboardFocus ? -1 : tabIndex,
```